### PR TITLE
Location permission on Map

### PR DESCRIPTION
**Pull request by Vanethos**
Add location permissions to map screen

### CODE REVIEW

File: lib/presentation/ui/home/home_page.dart [hunk 1]
```diff
@@ -264,7 +292,55 @@ class MapWidget extends StatelessWidget {
...
+            onTap: _requestPermission,
+            child: Icon(Icons.not_listed_location, size: size.shortestSide * .40,),
+          ),
+        ),
+        Container(
+            margin: EdgeInsets.only(top: marginMega),
+            alignment: Alignment(0, 0),
+            width: size.shortestSide * .7,
+            child: Text(
+              "This app needs the GPS location so that it can show where you are in this map, do you want to allow it?",
```
Comment (miguelpruivo) [inline lib/presentation/ui/home/home_page.dart:416]: Shouldn't this be on internalization support?

File: lib/presentation/ui/home/home_page.dart [hunk 2]
```diff
@@ -264,7 +292,55 @@ class MapWidget extends StatelessWidget {
...
+              style: styleMediumBlackText(),
+            )),
+        FlatButton(
+            padding: EdgeInsets.all(marginMedium),
+            onPressed: () {
+              _requestPermission();
+              Navigator.of(context).pop();
+              },
+            child: Text(
+              "Allow Location".toUpperCase(),
```
Comment (miguelpruivo) [inline lib/presentation/ui/home/home_page.dart:427]: The same.

File: lib/presentation/ui/home/home_page.dart [hunk 3]
```diff
@@ -264,7 +292,55 @@ class MapWidget extends StatelessWidget {
...
+
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.of(context).size;
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.start,
+      mainAxisSize: MainAxisSize.min,
+      crossAxisAlignment: CrossAxisAlignment.center,
+      children: <Widget>[
+        Container(child: Text("Permissions", style: styleIntroTitle(),)),
```
Comment (miguelpruivo) [inline lib/presentation/ui/home/home_page.dart:402]: Shouldn't this be on internalization support?

**Review by miguelpruivo — CHANGES_REQUESTED** [review]

**Comment by Vanethos** [comment]
Regarding the comment: you are right, but since we don't have yet the copy, I thought of opening an issue in the future about it

**Review by miguelpruivo — APPROVED** [review]
Internalization support will be added on the future.

**Pull request merged**